service/history/transferQueueActiveTaskExecutor.go

```diff
@@ -937,14 +947,6 @@ func (t *transferQueueActiveTaskExecutor) createFirstWorkflowTask(
 		Clock:               clock,
 	})
 
-	if err != nil {
-		if _, ok := err.(*serviceerror.NotFound); ok {
-			// Maybe child workflow execution already timed out or terminated
-			// Safe to discard the error and complete this transfer task
-			return nil
-		}
-	}
```

Comment:
I removed this override from here and few other places because this error bubble up to task error handler which already ignores `NotFound` and `NamespaceNotFound` errors.